bedrock/firefox/templates/firefox/private-browsing.html

```diff
@@ -0,0 +1,169 @@
+      </div>
+      <p class="footer">
+        <strong>
+          <a class="android-link" href="https://support.mozilla.org/en-US/kb/private-browsing-firefox-android">{{ _('Learn how to use Private Browsing') }}</a>
+          <a class="ios-link" href="https://support.mozilla.org/kb/private-browsing-firefox-ios">{{ _('Learn how to use Private Browsing') }}</a>
+          <a class="desktop-link" href="https://support.mozilla.org/en-US/kb/private-browsing-use-firefox-without-history">{{ _('Learn more about Private Browsing') }}</a>
+        </strong>
+        <small id="tracking-protection-caveat"> {{ _('Tracking Protection is only available with Firefox for Android and Desktop') }}</small>
+      </p>
+
+    </div>
+  </section>
+
+  <section class="when-to-use" id="when-to-use">
+    <div class="container">
+      <h2>{{ _('When to use Private Browsing') }}</h2>
+      <ul>
+        <li>
+          <h3>{{ _('To minimize your digital footprint') }}</h3>
+          <p>{{ _('Tracking Protection blocks Internet companies from following you across the Web') }}</p>
+        </li>
+        <li>
+          <h3>{{ _('Online banking from a shared device or computer') }}</h3>
+          <p>{{ _('Your username and password won’t be saved') }}</p>
+        </li>
+        <li>
+          <h3>{{ _('To help you plan a surprise, like a birthday or a trip') }}</h3>
+          <p>{{ _('No one can see your browsing history') }}</p>
+        </li>
+      </ul>
+
+    </div>
+  </section>
+
+  <section class="control-center" id="control-center">
+    <div class="container">
+      <h2>{{ _('The Control Center has you covered') }}</h2>
+      <p>
+        {{ _('When you see the shield to the left of the address bar, that means Firefox is protecting you from potential trackers.') }}
+        {{ _('You can disable Tracking Protection for a single site, or turn it off entirely. It’s up to you.') }}
+      </p>
+      <p class="footer">
+        <strong>
+          <a class="android-link" href="https://support.mozilla.org/kb/mixed-content-blocker-firefox-android">{{ _('Learn more about the Control Center') }}</a>
+          <a class="desktop-link" href="https://support.mozilla.org/kb/control-center-site-privacy-and-security-firefox">{{ _('Learn more about the Control Center') }}</a>
+        </strong>
+        <small id="control-center-caveat"> 
+        {{ _('Control Center is not available on Android devices, but a shield will still be displayed to let you know when you are protected.') }}</small>
+      </p>
+    </div>
+  </section>
+
+  <section class="cta" id="cta">
+    <div class="container">
+      <div class="download-buttons">
+        <div class="firefox-latest-cta">
+          <h2>{{ _('Try it now to stay in control') }}</h2>
+          <a href="https://www.mozilla.org" class="button-flat-dark" data-highlight="privateWindow" role="button">{{ _('Try Private Browsing') }}</a>
```

Comment:
This one should be a more appropriate fallback link too, like the other CTA above :+1: